packages/typespec-python/test/generated/azure-core-lro-standard/_specs_/azure/core/lro/standard/_operations/_operations.py

```diff
@@ -71,9 +71,12 @@ def build_standard_create_or_replace_request(name: str, **kwargs: Any) -> HttpRe
 
 
 def build_standard_delete_request(name: str, **kwargs: Any) -> HttpRequest:
+    _headers = case_insensitive_dict(kwargs.pop("headers", {}) or {})
```

Comment:
for a standard 202 initial response for a delete LRO, the service returns an `OperationStatus` body. Since it returns a body we should send an `accept` header